typescript/src/start_script.ts

```diff
@@ -18,6 +18,7 @@ import { execFileSync } from 'child_process';
 import * as path from 'path';
 import * as yargs from 'yargs';
 import * as fs from 'fs-extra';
+import { Z_FIXED } from 'zlib';
```

Comment:
this is clearly a gift from VS Code.  It sometimes inserts some random imports even before you realize that. Please remove it :)